src/mavis/annotate/file_io.py

```diff
@@ -344,9 +366,9 @@ class ReferenceFile:
     LOAD_FUNCTIONS: Dict[str, Optional[Callable]] = {
         'annotations': load_annotations,
         'reference_genome': load_reference_genome,
-        'masking': load_masking_regions,
+        'masking': load_known_sv,
```

Comment:
why are you changing the load masking regions function? that should not be affected by these changes

Reply:
`load_masking_regions` was moved to `load_known_sv` since the format of masking file is the same format as the `dgv` calls. It doesn't make sense to add redundant code just for a masking file. In the future, we might need to have a MAVIS formatted masking file in case users want to completely filter out certain calls without filtering out every call within regions.

Reply:
the difference being that the load_masking_regions function wouldn't be marked deprecated

Reply:
I've never actually used deprecated decorators before but it seems like we can just use the warnings lib

https://stackoverflow.com/questions/2536307/decorators-in-the-python-standard-lib-deprecated-specifically

Reply:
Addressed in [0e7aad3](https://github.com/bcgsc/mavis/pull/336/commits/0e7aad362a9cbf50adc2651df62a218dd698c7ff).